doc/en/customize.rst

```diff
@@ -48,6 +48,16 @@ Here is the algorithm which finds the rootdir from ``args``:
   directory. This allows to work with pytest in structures that are not part of
   a package and don't have any particular ini-file configuration.
 
+If no ``args`` are given, pytest collects test below the current working
+directory and also starts determining the rootdir from there. 
+
+:warning: custom pytest plugin commandline arguments may include a path, as in
+    ``py.test --log-output ../../test.log args``. Then ``args`` is mandatory,
```

Comment:
While you are at it, could you change `py.test` to `pytest`? We will be releasing 3.0 today probably, so it's fine for this to use the new ter from the start.